test/models/conclusion_final_review_test.rb

```diff
@@ -30,8 +30,8 @@ class ConclusionFinalReviewTest < ActiveSupport::TestCase
 
   # Prueba la creación de un informe final
   test 'create' do
-    Current.user = users :supervisor
-    review = Review.find reviews(:review_approved_with_conclusion).id
+    Current.user   = users :supervisor
+    review         = Review.find reviews(:review_approved_with_conclusion).id
```

Comment:
Esto quedó de antes, pero ya que tocamos la linea se puede cambiar por:
```suggestion
    review         = reviews :review_approved_with_conclusion
```